src/librustdoc/html/render.rs

```diff
@@ -814,16 +818,20 @@ themePicker.onclick = function() {{
                        .join(",")).as_bytes(),
     )?;
 
-    write(cx.dst.join(&format!("main{}.js", cx.shared.resource_suffix)),
-                      include_bytes!("static/main.js"))?;
-    write(cx.dst.join(&format!("settings{}.js", cx.shared.resource_suffix)),
-                      include_bytes!("static/settings.js"))?;
+    write_minify(cx.dst.join(&format!("main{}.js", cx.shared.resource_suffix)),
+                 include_bytes!("static/main.js"),
+                 enable_minification)?;
+    write_minify(cx.dst.join(&format!("settings{}.js", cx.shared.resource_suffix)),
+                 include_bytes!("static/settings.js"),
```

Comment:
Could we minimize these two `*.js` files at `build.rs` of rustdoc?

Reply:
I'd rather prefer that we don't. We *could* but then disabling minification would be a bit more complicated and I'm not sure it's worth it.